bitnami/spring-cloud-dataflow/Chart.yaml

```
@@ -53,4 +53,4 @@ maintainers:
 name: spring-cloud-dataflow
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/spring-cloud-dataflow
-version: 23.0.0
+version: 22.1.0
```

Bump the minor version properly

```suggestion
version: 23.1.0
```

---

fixed at https://github.com/bitnami/charts/pull/18749/commits/edc3ac67669f94c68712b8fe91c047ce6979a4e2